### DISCUSSION TITLE
Invalid: Input must fall within specified range

### DISCUSSION THREAD

**Discussion by kellytdunn** [FlowmapBlue/FlowmapBlue#77] [Q&A] [answered]
I get this error message in the google sheet when I enter any text or any number in the "origin" or "dest" column of greater than three digits of the flows tab. My origins and destination names are census tracts and three digits won't cut it. I tried ignoring it but nothing shows up on the map. Can anyone advise?

**Comment by ilyabo** [comment, marked as answer]
Hi Kelly, it might be that the column names are wrong.  Could you share the spreadsheet (ilyabo AT gmail DOT com)?

**Reply by kellytdunn** [reply]
Just shared with you. Thanks!

**Reply by kellytdunn** [reply]
For anyone else who is following: The id column values have to match the values referenced in origin and dest columns.
